src/main/webapp/app/services/visualizations/visualization.directive.js

```diff
@@ -29,14 +29,15 @@
         };
         return directive;
 
-        function link(scope, element, attrs) {}
+        function link(scope, element, attrs) { }
     }
 
     VisualizationRenderController.$inject = [
         '$scope',
         'GenerateStackedverticalbarChart',
         'GenerateStackedhorizontalbarChart',
         'GenerateClusteredverticalbarChart',
+        'clusteredverticalbar',
```

Comment:
i think you can keep the same name for service as before so that u wont need to add new service name?

Reply:
GenerateClusteredverticalbarChart and clusteredverticalbar both are used
inside GenerateClusteredverticalbarChart I have use clusteredverticalbar service 
we nedd to use GenerateClusteredverticalbarChart service becuase its use other of of flair-bi application for processing data